ddl/partition.go

```diff
@@ -812,6 +812,17 @@ func (w *worker) onExchangeTablePartition(d *ddlCtx, t *meta.Meta, job *model.Jo
 	// exchange table meta id
 	partDef.ID = nt.ID
 
+	// Clear the tiflash replica available status.
```

Comment:
should remove this comment since the code no longer clear the tiflash replica available status